boards.txt

```diff
@@ -19779,7 +19779,7 @@ AirM2M_CORE_ESP32C3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 XIAO_ESP32C3.name=XIAO_ESP32C3
 XIAO_ESP32C3.vid.0=0x2886
-XIAO_ESP32C3.pid.0=0x0047
+XIAO_ESP32C3.pid.0=0x0046
```

Comment:
Just to be sure, is there any reason to change the PID of XIAO_ESP32C3?

Reply:
The previous PID was incorrect.